### PR TITLE
document stringData for secrets

### PR DESCRIPTION
**Pull request by bparees**
@liggitt ptal

### CODE REVIEW

**Comment by bparees** [comment]
also what was the first release that supported stringData?  3.3?

**Comment by bparees** [comment]
@deads2k are you able to review this?

**Comment by deads2k** [comment]
> also what was the first release that supported stringData? 3.3?

Not in 3.3.  New in 3.4: https://github.com/openshift/ose/blob/enterprise-3.3/vendor/k8s.io/kubernetes/pkg/api/types.go#L2605

File: dev_guide/secrets.adoc [hunk 1]
```diff
@@ -30,13 +30,19 @@ metadata:
...
   password: "dmFsdWUtMg0KDQo="
+stringData: <3>
+  hostname: "myapp.mydomain.com" <4>
 ----
 <1> The allowable format for the keys in the `data` field must meet the
 guidelines in the *DNS_SUBDOMAIN* value in
 https://github.com/GoogleCloudPlatform/kubernetes/blob/v1.0.0/docs/design/identifiers.md[the
 Kubernetes identifiers glossary].
+<2> The value associated with keys in the the `*data*` map must be base64 encoded.
+<3> Entries in the `*stringData*` map will have their values converted to base64 and the
```
Comment (deads2k) [inline dev_guide/secrets.adoc]: Note that this write-only.  The API never returns it.

**Comment by deads2k** [comment]
one comment. lgtm otherwise.

File: dev_guide/secrets.adoc (hunk 1 above)
Comment (bparees) [inline dev_guide/secrets.adoc]: thanks amended to clarify.

**Comment by bparees** [comment]
@openshift/team-documentation ptal/merge.

**Comment by ahardin-rh** [comment]
LGTM. Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../dev_guide/secrets.adoc#dev-guide-secrets[Secrets]
|Added details about `*stringData*` for secrets.
%

**Pull request merged**